<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head><meta http-equiv="Content-Type" content="text/html;charset=iso-8859-1">
<title>LiteSQL: litesql::InternalError Class Reference</title>
<link href="doxygen.css" rel="stylesheet" type="text/css">
<link href="tabs.css" rel="stylesheet" type="text/css">
</head><body>
<!-- Generated by Doxygen 1.4.5 -->
<div class="tabs">
  <ul>
    <li><a href="index.html"><span>Main&nbsp;Page</span></a></li>
    <li id="current"><a href="annotated.html"><span>Classes</span></a></li>
    <li><a href="files.html"><span>Files</span></a></li>
    <li><a href="dirs.html"><span>Directories</span></a></li>
    <li><a href="pages.html"><span>Related&nbsp;Pages</span></a></li>
  </ul></div>
<div class="tabs">
  <ul>
    <li><a href="annotated.html"><span>Class&nbsp;List</span></a></li>
    <li><a href="hierarchy.html"><span>Class&nbsp;Hierarchy</span></a></li>
    <li><a href="functions.html"><span>Class&nbsp;Members</span></a></li>
  </ul></div>
<div class="nav">
<b>litesql</b>::<a class="el" href="classlitesql_1_1InternalError.html">InternalError</a></div>
<h1>litesql::InternalError Class Reference</h1><!-- doxytag: class="litesql::InternalError" --><!-- doxytag: inherits="litesql::Except" -->exception thrown when backend produces internal error  
<a href="#_details">More...</a>
<p>
<code>#include &lt;<a class="el" href="except_8hpp-source.html">except.hpp</a>&gt;</code>
<p>
Inheritance diagram for litesql::InternalError:<p><center><img src="classlitesql_1_1InternalError__inherit__graph.png" border="0" usemap="#litesql_1_1InternalError__inherit__map" alt="Inheritance graph"></center>
<map name="litesql_1_1InternalError__inherit__map">
<area href="classlitesql_1_1Except.html" shape="rect" coords="25,9,129,33" alt="">
</map>
<center><font size="2">[<a href="graph_legend.html">legend</a>]</font></center>Collaboration diagram for litesql::InternalError:<p><center><img src="classlitesql_1_1InternalError__coll__graph.png" border="0" usemap="#litesql_1_1InternalError__coll__map" alt="Collaboration graph"></center>
<map name="litesql_1_1InternalError__coll__map">
<area href="classlitesql_1_1Except.html" shape="rect" coords="25,9,129,33" alt="">
</map>
<center><font size="2">[<a href="graph_legend.html">legend</a>]</font></center><a href="classlitesql_1_1InternalError-members.html">List of all members.</a><table border="0" cellpadding="0" cellspacing="0">
<tr><td></td></tr>
<tr><td colspan="2"><br><h2>Public Member Functions</h2></td></tr>
<tr><td class="memItemLeft" nowrap align="right" valign="top"><a class="anchor" name="7fd142bfa1d85ab5e9e548bb1b3f71b3"></a><!-- doxytag: member="litesql::InternalError::InternalError" ref="7fd142bfa1d85ab5e9e548bb1b3f71b3" args="(std::string m)" -->
&nbsp;</td><td class="memItemRight" valign="bottom"><b>InternalError</b> (std::string m)</td></tr>

</table>
<hr><a name="_details"></a><h2>Detailed Description</h2>
exception thrown when backend produces internal error 
<p>
<hr>The documentation for this class was generated from the following file:<ul>
<li><a class="el" href="except_8hpp-source.html">except.hpp</a></ul>
<hr />
<a href="http://sourceforge.net"><img src="http://sflogo.sourceforge.net/sflogo.php?group_id=113304&amp;type=2" width="125" height="37" border="0" alt="SourceForge.net Logo" /></a>

